src/components/AppLoading.tsx

```diff
@@ -25,6 +28,41 @@ export const AppLoading: FC<PropsWithChildren> = ({ children }) => {
 
   const isLoading = !isLoadingComplete || isSignedIn === null || !isLayoutReady
 
+  const currentVersion = Application.nativeApplicationVersion || 'unknown'
+  const os = Platform.OS
+
+  const { mutate: checkForUpdate } = useSystemControllerCheckForAppUpdate()
```

Comment:
Add isLoading from mutation to isLoading on 29 line

Reply:
Gonna add this, thank you 🤓